### ISSUE TITLE
Vertical Spacing in \cup<tab>\rightharpoonaccent<tab>

### ISSUE THREAD

**Issue by goretkin** [cormullion/juliamono#99]
on macOS in VSCode:
![image](https://user-images.githubusercontent.com/581043/109259037-6e718f80-77c9-11eb-99c3-ca28ed49870e.png)

Can there be a bit more space between the cup and the arrow?

**Comment by cormullion** [comment]
At first glance, this looks to be https://cormullion.github.io/pages/2020-07-26-JuliaMono/#why_dont_these_accentsmarks_work_properly .

For example, in Terminal (macOS):

<img width="102" alt="Screenshot 2021-02-26 at 10 58 00" src="https://user-images.githubusercontent.com/52289/109291918-837a0d00-7821-11eb-8f0d-0029a84cfe02.png">

and iTerm2 (macOS)

<img width="121" alt="Screenshot 2021-02-26 at 10 56 07" src="https://user-images.githubusercontent.com/52289/109291854-68a79880-7821-11eb-9c21-ada0190b993b.png">

and CotEditor (useful free editor with good font support, macOS)

<img width="89" alt="Screenshot 2021-02-26 at 10 58 52" src="https://user-images.githubusercontent.com/52289/109292008-a3113580-7821-11eb-8493-f05392a13d32.png">

It's a bit disappointing that this looks bad in both VSCode's Terminal (the xterm.js emulator) _and_ the main VSCode's editor window.

<img width="396" alt="Screenshot 2021-02-26 at 11 00 49" src="https://user-images.githubusercontent.com/52289/109292193-e79cd100-7821-11eb-92f1-365d4e3e8080.png">

That said, although I don't think I can fix the xterm.js display issues, I could perhaps try to persuade the more powerful text engines inside VSCode and Atom to do a bit better...

**Comment by cormullion** [comment]
Yes. I've just changed all the maths glyphs in U+2200 to U+22ff to have suitable definitions for top marks:

<img width="610" alt="Screenshot 2021-02-26 at 11 30 26" src="https://user-images.githubusercontent.com/52289/109295078-0b621600-7826-11eb-90a2-44a46a83b304.png">

Will be in the next release.

**Comment by goretkin** [comment]
Thanks so much for the rapid response! I'm curious to know what the change involved, but I did not find a corresponding commit in this repository.

**Comment by cormullion** [comment]
No, I haven't compiled the font or uploaded it yet... 😄I made a release just two days ago, so it's a bit soon for another one.

The change I made was to add ```mark anchors``` to glyphs that don't normally have them. 

This is what I see now, a small brown anchor point that wasn't there before. This apparently helps apps like VSCode and Atom (that struggle with mark placement usually) do a better job. It seems that, on macOS,  Terminal and iTerm do a better job of placing marks in the right (obvious) place, and don't need this suggestion. 

<img width="302" alt="Screenshot 2021-02-26 at 17 54 31" src="https://user-images.githubusercontent.com/52289/109336781-ca84f400-785b-11eb-9a1c-a8920cbc62d4.png">
